src/main/java/school/lemon/changerequest/java/matrix/Matrix.java

```diff
@@ -0,0 +1,117 @@
+package school.lemon.changerequest.java.matrix;
+
+import java.util.Random;
+
+public class Matrix {
+    private int value[][];
```

Comment:
Does `int` type cover any `Real` number? Can I put `3.14` into `int`? You were asked to create 2d array of real numbers.